app/controllers/extensions_controller.rb

```diff
@@ -349,8 +350,8 @@ def qualify_scope(scope, params)
       scope = scope.featured
     end
 
-    if params[:tier].present?
-      tier = Tier.find_by(name: params[:tier])
+    if params[:tiers].present?
+      tier = Tier.find_by(id: params[:tiers])
```

Comment:
This line won't work if multiple tiers are selected.  Only one of the tiers will survive the `.find_by` call.

Also, we have to allow for the fact that if an extension's `tier_id` is nil, the extension belongs to the default tier.

All together, the block of code needs to be:
```
    if params[:tiers].present?
      tier_targets = params[:tiers]
      tier_targets << nil if tier_targets.map(&:to_s).include?(Tier.default.id.to_s)
      scope = scope.where(raw_tier: tier_targets)
    end
```

Reply:
Fixed scope for multiple selections